plugins/@grouparoo/pardot/src/lib/export/destinationMappingOptions.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  DestinationMappingOptionsMethod,
+  DestinationMappingOptionsResponseProperty,
+  DestinationMappingOptionsResponseTypes,
+} from "@grouparoo/core";
+
+import PardotClient from "../client";
+import { connect } from "./../connect";
+
+// Prospect field spec https://developer.pardot.com/kb/object-field-references/#prospect
+const requiredProspectFields: DestinationMappingOptionsResponseProperty[] = [
+  { key: "email", type: "email" },
+];
+
+const defaultProspectFields: DestinationMappingOptionsResponseProperty[] = [
+  { key: "campaign_id", type: "integer" }, // TODO check for validation
+  { key: "salutation", type: "string" },
+  { key: "first_name", type: "string", important: true },
+  { key: "last_name", type: "string", important: true },
+  { key: "password", type: "string" },
```

Comment:
is this for real?

Reply:
![image](https://user-images.githubusercontent.com/4368928/109698671-07a3fb80-7b66-11eb-9ed1-c821d74bff3a.png)
Yup!